app/lib/admin/models.dart

```diff
@@ -189,6 +189,8 @@ class ModerationSubject {
   }
 
   late final fqn = '$kind:$localName';
+  bool get hasPackage => package != null;
+  bool get hasPublisher => publisherId != null;
```

Comment:
```suggestion
  bool get isPublisher => publisherId != null;
```

Reply:
Less, sure how `hasPackage` should be named :D

Reply:
Renamed to `isPackage` to keep it consistent.